pydarn/utils/coordinates.py

```diff
@@ -172,7 +174,7 @@ def gate2geographic_location(stid: int, beam: int, height: float = None,
         beam_edge = 0
 
     # psi [rad] in the angle from the boresight
-    psi = beam_sep * (beam - offset) + beam_edge
+    psi = beam_sep * (beam - offset) - beam_edge + bmoff
```

Comment:
Why was the sign changed from `+ beam_edge` to `- beam_edge` ?

Reply:
The sign change was suggested by Colin in this comment:
https://github.com/SuperDARN/pydarn/issues/351#issuecomment-1666645158

Reply:
@carleyjmartin oh ok thanks, now I see that suggested change.  This has the practical effect of shifting all of the beams eastward by one, eg printing out the `psi` values for the CVW radar for each beam:

```
       0      -37.260000      -38.880000      -35.640000
       1      -34.020000      -35.640000      -32.400000
       2      -30.780000      -32.400000      -29.160000
       3      -27.540000      -29.160000      -25.920000
       4      -24.300000      -25.920000      -22.680000
       5      -21.060000      -22.680000      -19.440000
       6      -17.820000      -19.440000      -16.200000
       7      -14.580000      -16.200000      -12.960000
       8      -11.340000      -12.960000      -9.7200000
       9      -8.1000000      -9.7200000      -6.4800000
      10      -4.8600000      -6.4800000      -3.2400000
      11      -1.6200000      -3.2400000       0.0000000
      12       1.6200000       0.0000000       3.2400000
      13       4.8600000       3.2400000       6.4800000
      14       8.1000000       6.4800000       9.7200000
      15       11.340000       9.7200000       12.960000
      16       14.580000       12.960000       16.200000
      17       17.820000       16.200000       19.440000
      18       21.060000       19.440000       22.680000
      19       24.300000       22.680000       25.920000
      20       27.540000       25.920000       29.160000
      21       30.780000       29.160000       32.400000
      22       34.020000       32.400000       35.640000
      23       37.260000       35.640000       38.880000
```

where the first column is beam number, the second column is for the beam center, the third column is using `+beam_edge`, and the fourth column is `-beam_edge`.  You can see that the beam edges in the last column no longer correspond to the "left" beam edge, which is the standard convention.  Maybe this is hemisphere specific?

Reply:
Oooo okay, It could possibly be a fix for the -ve beam_sep valued radars, I think they're all in the southern hemisphere.

I'll figure it out!